src/main/java/com/google/sps/data/RoomState.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.data;
+
+public class RoomState {
+
+    public static VideoState currentState;
+    public static long currentVideoTimestamp;
+    public static int currentVideo;
+    public static int messageCount;
+    //Default RoomState constructor
+    public RoomState() {
+        this.currentState = VideoState.UNSTARTED;
+        this.currentVideoTimestamp = 0;
+        this.currentVideo = 0;
+        this.messageCount = 0;
+    }
```

Comment:
Normally we don't want a default constructor. If this is used when first creating a room, I think it would be better to use the regular constructor.

Create static fields here to represent empty values for currentVideoTimestamp, currentVideo, and messageCount